src/librbd/api/Mirror.cc

```diff
@@ -134,6 +136,135 @@ struct C_ImageGetStatus : public C_ImageGetInfo {
   }
 };
 
+int map_instance_id_to_service_id(librados::IoCtx& io_ctx,
```

Comment:
I'm not sure I really like the idea of each API call issuing a ```service status``` dump to the mgr. The rbd CLI can do this one and the dashboard will have all this data available in-memory. Therefore, perhaps the API should just return the daemon's pool instance id and leave it up to the caller to map it back to a service?

Reply:
I didn't like this much too but wanted the API methods to do as mach as possible so we would not duplicate this in cli and dashboard. But after your argument about dashboard already having this data available in-memory I tend to agree, so I will change this just to return the daemon's pool instance id.